src/control/Control.Zoom.js

```diff
@@ -81,7 +81,7 @@ export var Zoom = Control.extend({
 
 	_createButton: function (html, title, className, container, fn) {
 		var link = DomUtil.create('a', className, container);
-		link.innerHTML = html;
+		link.innerHTML = '<span aria-hidden="true">' + html + '</span>';

```

Comment:
Adding another element here might break plugins that are extending this component and expecting a certain DOM structure.

Reply:
What if we change the default text option?

https://github.com/Leaflet/Leaflet/blob/5cd4afe80cbd64ba6ef25d2d53376e1c5aeefd14/src/control/Control.Zoom.js#L23

`zoomInText: '<span aria-hidden="true">+</span>', `

Reply:
Even though it still adds an additional element I think this might be the best option. Let's do this @Falke-Design